views/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Personal Portfolio</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../styles/styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins&display=swap" rel="stylesheet">
+
+<body>
+    <nav class="toolbar">
+        <img src="../images/image_geometry_1.png" alt="navbar image">
+        <img class="navbar-image" src="../images/ic_menu.svg" alt="navbar menu">
```

Comment:
- [ ] [REQUIRED] Please enclose the hamburger button in a `<nav>` tag because the nav tag is reserved for primary navigation areas, like the main menu across the top of the page or section.😊😊